Dockerfile

```diff
@@ -123,7 +124,9 @@ RUN ln -s /usr/bin/python3 /usr/bin/python \
     awscli==${AWS_CLI_VERSION} \
     'colorama<0.4.0,>=0.3.9' \
     'urllib3<1.25,>=1.24.1' \
- && aws --version
+    docker-compose \
```

Comment:
Changed docker-compose to pip install. See https://docs.docker.com/compose/install/#install-using-pip
Note the added apk dependencies